feat: add li to ul

yoshikawa-akr opened the pull request:
# [issue No.1](https://github.com/kenmori/handsonFrontend/blob/master/work/markup/1.md#1%E3%81%93%E3%81%AEdom%E3%82%92html%E5%86%85%E3%81%AEul%E3%81%AE%E4%B8%AD%E3%81%AB%E5%B7%AE%E3%81%97%E8%BE%BC%E3%82%93%E3%81%A7%E3%81%8F%E3%81%A0%E3%81%95%E3%81%84)

**Insert this DOM into the ul in the html.**

## [CodeSandBox](https://codesandbox.io/s/polished-silence-7q377) 
[Latest commit](https://codesandbox.io/s/wizardly-spence-hwy7g?file=/index.html)

## In terms of implementation, what you are worried about, what you want to focus on, and what you want to ask.
- Is there a better way to write?

yoshikawa-akr committed:
feat: add li to ul

Create a new js file and add the code.

443n0511 commented on issue_01/index.html:
```diff
@@ -14,5 +14,6 @@
 </body>
+<script src="script.js"></script>
 
 </html>
```
MUST
Please make a new line at the end of the sentence.
You might want to review your editor settings.

[コードの行末を改行してないとGithubで注意されるのは、editorの設定で防ぐ✏️【Android Studio】](https://qiita.com/syuheifujita/items/960754d0c821f9f6407c)

yoshikawa-akr replied:
Thank you.
I have made the correction based on the article you attached.
[a14cf96](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/a14cf965696798ecef1d35bcfd09cf264d83369a)

443n0511 replied:
Excellent! That's perfect!!!!

443n0511 commented on issue_01/script.js:
```diff
@@ -0,0 +1,4 @@
+const ul = document.getElementsByTagName('ul');
```
IMO
If the element to be retrieved is predetermined, it is ideal to give it an id and retrieve it.

yoshikawa-akr replied:
Thank you.
I have given the ul element an id and changed the method.
[5e283ad](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/5e283ad7096e5d6acd94d6e58a649be06b61d0e0)

443n0511 commented on issue_01/script.js:
```diff
@@ -0,0 +1,4 @@
+const ul = document.getElementsByTagName('ul');
+const li = document.createElement("li");
+li.textContent = "これです"
+ul.item(0).appendChild(li);
```
IMO
If you get the id instead of `getElementsByTagName`, you can also `appendChild` without specifying the index. Please try it.

yoshikawa-akr replied:
Thank you.
By using `getElementById`, I was able to get rid of the index specification!
[1247e4b](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/1247e4bfbe19e7ef2a962b497bdc82802c780743)

kanakonagiri commented:
I agree with [443n0511](https://github.com/443n0511) san's comments.
Try using [Document.getElementById()](https://developer.mozilla.org/ja/docs/Web/API/Document/getElementById).
Then, you can quickly access specific elements, and you don't have to specify the index.

yoshikawa-akr committed:
style: Add a new line at the end of the line.

Change the editor settings.

yoshikawa-akr committed:
refactor: Change method.

Give the ul element an id and get it with getElementById.

yoshikawa-akr committed:
refactor: Remove the description of index.

Removed the index description by giving it an id and using getElementById to get the ul element.

yoshikawa-akr commented:
>

yoshikawa-akr commented:
> I agree with [443n0511](https://github.com/443n0511) san's comments. Try using [Document.getElementById()](https://developer.mozilla.org/ja/docs/Web/API/Document/getElementById). Then, you can quickly access specific elements, and you don't have to specify the index.

@[kanakonagiri](https://github.com/kanakonagiri)
Thanks for the comment!
I was able to get rid of the index specification by giving the ul element an id and retrieving it with `getElementId`.
[1247e4b](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/1247e4bfbe19e7ef2a962b497bdc82802c780743)

443n0511 commented on issue_01/index.html:
```diff
@@ -11,8 +11,9 @@
 <body>
     <div>
-        <ul></ul>
+        <ul id="target"></ul>
```
IMO
IDs and classes that are the starting point for JavaScript are prefixed with `js-` to clearly indicate that JavaScript is involved.
It is also a good idea to use an ID name that is easy for everyone to understand. This time, why not keep it simple and use ul?
[Reference](https://zenn.dev/haniwaman/articles/bf392f397c8db7341881#javascript%E3%81%A8%E7%B4%90%E3%81%A5%E3%81%8F%E8%A6%81%E7%B4%A0%E3%81%AFjs-%E3%81%AE%E6%8E%A5%E9%A0%AD%E8%BE%9E)
```suggestion
        <ul id="js-ul"></ul>
```

yoshikawa-akr replied:
Thank you!
You're right, the ID should be easy for everyone to understand.
 I will be aware of this in the future.
[7338b46](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/7338b462283954623d7f947002b6f7418a9b21c8)

yoshikawa-akr committed:
refactor: Changing the id name.

Change the id of the ul element from target to js-ul.
